### PR TITLE
Revert use shellwords to escape args and fix

### PR DESCRIPTION
**Pull request by ransombriggs**
Reverts https://github.com/heroku/heroku-run/commit/5875bc2a6df098d758c734fdc1c788e472321967 and includes logic to fix issues around double quotes in args.

### CODE REVIEW

**Comment by tcrayford** [comment]
what was the issue with double quotes in args?

**Comment by ransombriggs** [comment]
@tcrayford See https://github.com/heroku/heroku-run/pull/10 for the issue

**Comment by mathias** [comment]
I like the tests. 

Seems a little hard to follow everything it is doing without stepping through it in my head, but since this is mostly restoring (reverting) what code was there before, I'm fine it with.

LGTM.

**Comment by rsampaio** [comment]
:+1:
